pkg/agent/qrm-plugins/network/staticpolicy/policy.go

```diff
@@ -99,6 +99,9 @@ func NewStaticPolicy(agentCtx *agent.GenericContext, conf *config.Configuration,
 	// it is incorrect to reserve bandwidth on those diabled NICs.
 	// we only count active NICs as available network devices and allocate bandwidth on them
 	enabledNICs := filterNICsByAvailability(agentCtx.KatalystMachineInfo.ExtraNetworkInfo.Interface, nil, nil)
+	if len(enabledNICs) == 0 {
+		return false, agent.ComponentStub{}, fmt.Errorf("no available nics on this node")
```

Comment:
it may cause the plugin restarts repeatedly,  if we should make adaption when enabledNICs is empty instead of returning error?